drafts/prometheus-alertmanager.adoc

```diff
@@ -0,0 +1,250 @@
++
+. In the `prometheus.yml` file, add the location of your `alert.yml` file under `rule_files`. If it's not in the same directory as Prometheus, make sure you supply the full path.
+. Restart Prometheus. Navigate to `http://localhost:9090` and click on `Alerts` on the navigation bar. You should be able to see your alert.
+
+image::/img/blog/prometheusAM_ui_alerts.png[Prometheus Alerts on UI, align="left"]
+
+== Configure Prometheus Alertmanager to pass alerts to Slack
+
+JENNIFER:  Add a brief introduction here to say what the reader will achieve/learn from this section (configure Prometheus Alertmanager to work with Slack and then  xxxx). 
+
+
+=== Configuring the Prometheus Alertmanager to communicate with Slack
+
+Now that Prometheus is set up with rules for our Liberty server, we can set up the Prometheus Alertmanager to connect to Slack.
+
+. Edit `alertmanager.yml` to add your Slack webhook under `global`.
++
+```
+  global:
+    slack_api_url: <your api url>
+```
++
+. Under the `route`, you will see that there is a receiver called `web.hook`. We will be editing this receiver to work with Slack. Right under `webhook_configs`, add another field (aligned with `webhook_configs`) called `slack_configs`. It'll look something like this:
++
+```
+slack_configs:
+- channel: 'prometheus-alertmanager-test'
+  title: "{{ range .Alerts }}{{ .Annotations.summary }}\n{{ end }}"
+  text: "*Description*: {{ .CommonAnnotations.description }}\n*Severity*: {{ .CommonLabels.severity }}""
+```
++
+The `CommonAnnotations` come from your `alert.yml` file. The text is written using the link:https://golang.org/pkg/text/template/[Go templating] system.
++
+. Run `./alertmanager`.
+
+The Alertmanager is located at `http://localhost:9093` by default. Since we haven't generated any test data yet, you won't see any alert groups at the moment. We'll do that next.
+
+=== Receiving alerts via Prometheus Alertmanager
+Now that Alertmanager is set up, we need to configure Prometheus to talk to it.
+
+. In the `prometheus.yml` file inside your Prometheus folder, add `localhost:9093` to the targets under the Alertmanager configuration:
++
+```
+# Alertmanager configuration
+alerting:
+  alertmanagers:
+  - static_configs:
+    - targets:
+        - localhost:9093
+```
++
+. Restart Prometheus.
+. Trigger the alert however you can. With the rule we're using, the easiest way is to change your threshold value to be very low and run your application a few times to use up some of your heap.
+. Head over to `http://localhost:9090` and click on **Alerts**. Your alert should be in the `Pending` or `Firing` state. Once the alert is in the `Firing` state, you should also be able to see your alert at `http://localhost:9093`.
+. Check your Slack channel to see your message.
+
+JENNIFER: Are there any obvious mistakes someone could make but might not realise that could stop this working?
```

Comment:
I don't think there's any obvious mistakes the reader could make - since the whole config is available in my repo, the only part that may cause issues is the Slack webhook. - Could add a sentence about making sure that the webhook works, but not sure if necessary.

ADD: If you're having trouble getting alerts to send to Slack, make sure that your webhook is working properly.

Reply:
Okay, it doesn't sound like it's necessary here then. Thanks